frontend/src/components/orgFacilities/ManageFacilityDetails.vue

```diff
@@ -143,11 +248,72 @@ export default {
       default: true,
     },
   },
+  emits: ['facility-updated'],
+  data() {
+    return {
+      editing: {
+        phone: false,
+        email: false,
+      },
+      workingFields: {
+        email: '',
+        phone: '',
+      },
+      valid: {
+        email: true,
+        phone: true,
+      },
+      facilityCopy: {
+        phone: '',
+        email: '',
+      },
+      isProcessing: false,
+      rules,
+    };
+  },
   computed: {
     ...mapState(useOrganizationStore, ['organizationProviderType']),
     providerType() {
       return capitalize(this.organizationProviderType);
     },
+    workingFieldInUse() {
+      return Object.values(this.editing).some((value) => value);
+    },
+  },
+  mounted() {
+    this.workingFields = { ...this.facility };
```

Comment:
There's a lot of unnecessary fields if you spread vs just destructuring phone and email.

Reply:
Yep, changes committed.